docs/pages/kubernetes-access.mdx

```diff
@@ -1,460 +1,28 @@
 ---
-title: Kubernetes Access Guide
-description: How to set up and configure Teleport for Kubernetes access with SSO and RBAC
-h1: Teleport Kubernetes Access
+title: Kubernetes Access
+description: Teleport Kubernetes Access introduction, demo and resources
 ---
 
-Teleport handles SSO and provides a unified access plane for Kubernetes clusters.
+Teleport provides a unified access for Kubernetes clusters.
 
-* Users can login once using [`tsh login`](cli-docs.mdx#tsh-login) and
-  switch between multiple clusters using [`tsh kube login`](cli-docs.mdx#tsh-kube-login).
-* Admins can use roles to implement policies like `developers must not access production`.
-* Achieve compliance by capturing `kubectl` events and session recordings for `kubectl exec`.
+* Users can receive short-lived certificates using Single Sign-On and switch between clusters without relogins.
+* Admins can use roles to implement policies "developers must not access production" and enforce
+dual authorization using [access workflows](./enteprise/workflow.md) for privileged operations.
+* Organizations can achieve compliance by capturing `kubectl` events and session recordings for `kubectl`.
 
-## Teleport Kubernetes Service
+## SSO and Audit for Kubernetes Clusters
 
-By default, the Kubernetes integration is turned off in Teleport. The configuration
-setting to enable the integration in the proxy service section in the `/etc/teleport.yaml`
-config file, as shown below:
+Set up single sign on, capture audit events and sessions with Teleport running in a Kubernetes cluster.
 
-```yaml
-# snippet from /etc/teleport.yaml on the Teleport proxy service:
-kubernetes_service:
-    enabled: yes
-    public_addr: [k8s.example.com:3027]
-    listen_addr: 0.0.0.0:3027
-    kubeconfig_file: /secrets/kubeconfig
-```
-Let's take a closer look at the available Kubernetes settings:
+<video muted playsInline controls>
+  <source src="https://goteleport.com/teleport/videos/kubernetes-access/kubelogin.mp4" type="video/mp4" />
+  <source src="https://goteleport.com/teleport/videos/kubernetes-access/kubelogin.webm" type="video/webm" />
+Your browser does not support the video tag.
+</video>
 
-- `public_addr` defines the publicly accessible address which Kubernetes API clients
-  like `kubectl` will connect to. This address will be placed inside of kubeconfig on
-  a client's machine when a client executes tsh login command to retrieve its certificate.
-  If you intend to run multiple Teleport proxies behind a load balancer, this must
-  be the load balancer's public address.
+## Getting Started
 
-- `listen_addr` defines which network interface and port the Teleport proxy server
-  should bind to. It defaults to port 3026 on all NICs.
+Configure Kubernets Access in a 10 minute [Getting Started](./kubernetes-access/getting-started.mdx) guide.
 
```

Comment:
Might also make sense to include links to guides/access control sections on this page as well?

Reply:
I'd like to focus readers attention on one thing here, as the most important CTA after intro is to get started